### PR TITLE
Specify Red Hat Satellite Client repo version

### PR DESCRIPTION
**Pull request by adamlazik1**
Specified Red Hat Satellite Client repo version in the *Synchronizing the New Repositories* procedure in *Upgrading and Updating* guide.
See [BZ#2119134](https://bugzilla.redhat.com/show_bug.cgi?id=2119134) for reference.

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.4/Katello 4.6
* [x] Foreman 3.3/Katello 4.5
* [x] Foreman 3.2/Katello 4.4
* [x] Foreman 3.1/Katello 4.3

### CODE REVIEW

File: guides/doc-Upgrading_and_Updating/topics/synchronizing_the_new_repositories.adoc [hunk 1]
```diff
@@ -31,6 +31,7 @@ In the {ProjectWebUI}, navigate to *Content* > *Subscriptions*, click *Manage Ma
 . In the {ProjectWebUI}, navigate to *Content* > *Sync Status*.
 . Click the arrow next to the product to view the available repositories.
 . Select the repositories for {ProjectVersion}.
+Note that *Red{nbsp}Hat {project-client-name}* does not have a {ProjectVersion} version; simply choose *Red{nbsp}Hat {project-client-name}*.
```
Comment (maximiliankolb) [inline guides/doc-Upgrading_and_Updating/topics/synchronizing_the_new_repositories.adoc:34]: I'd probably reword `; simply`, but your call.

Comment (adamlazik1) [reply guides/doc-Upgrading_and_Updating/topics/synchronizing_the_new_repositories.adoc]: Thanks for the suggestion! I reworded it a bit, see if you like this version better.

**Review by maximiliankolb — APPROVED** [review]
LGTM

**Comment by maximiliankolb** [comment]
Merged to master and cherry-picked: ✅ 
   2a9fb6404..295377875  3.4 -> 3.4
   29d45eaa0..1290ae2d9  3.3 -> 3.3
   36e1b10f3..dd1ededd3  3.2 -> 3.2
   eca55ac2e..b1fc4b0bc  3.1 -> 3.1

**Pull request merged**